lib/utils.pm

```diff
@@ -2371,16 +2372,30 @@ This functions checks if ca-certificates-suse is installed and if it is not it a
 
 sub ensure_ca_certificates_suse_installed {
     return unless is_sle || is_sle_micro;
+    my $ret = 0;
     if (script_run('rpm -qi ca-certificates-suse') == 1) {
         my $host_version = get_var("HOST_VERSION") ? 'HOST_VERSION' : 'VERSION';
         my $distversion = get_required_var($host_version) =~ s/-SP/_SP/r;    # 15 -> 15, 15-SP1 -> 15_SP1
-        zypper_call("ar --refresh http://download.suse.de/ibs/SUSE:/CA/SLE_$distversion/SUSE:CA.repo");
-        if (is_sle_micro) {
-            transactional::trup_call('--continue pkg install ca-certificates-suse');
+
+        # exit codes defined will not stop on error.
+        my $out = zypper_call("ar --refresh http://download.suse.de/ibs/SUSE:/CA/SLE_$distversion/SUSE:CA.repo", retry => 3, exitcode => [0, 1, 2, 3, 4]);
+        if ($out == 0) {
```

Comment:
I really don't like it that we're trying to workaround infra issues here. This makes the test flow unpredictable.

Can you justify why this is needed? We use this all over the place and I'm not aware on infra issues on such a scale that would justify this addition.